server/crashmanager/tests/test_signatures.py

```diff
@@ -88,111 +39,13 @@ def test_signatures_no_login(client, name, kwargs):
-    assert siglist[0].size == 1
-    assert siglist[1] == bucket1
-    assert siglist[1].size == 2
-    assert_contains(response, "bucket #1")
-    assert_contains(response, "bucket #2")
-    assert_contains(response, SIGS_FMT % 2)
-
-
-def test_all_signatures_view_logged(client, cm):  # pylint: disable=invalid-name
-    """Create a bucket, mark logged, and see that /all/ still shows the entry."""
-    client.login(username='test', password='test')
-    bucket = cm.create_bucket(shortDescription="bucket #1", bug=cm.create_bug('123'))
-    response = client.get(reverse("crashmanager:allsignatures"))
-    LOG.debug(response)
-    assert response.status_code == requests.codes['ok']
-    assert_contains(response, "bucket #1")
-    siglist = response.context['siglist']
-    assert len(siglist) == 1  # 1 bucket
-    assert siglist[0] == bucket  # same bucket we created
-    assert_contains(response, ALL_SIGS_FMT % 1)
-
-
-def test_all_signatures_view_logged_unlogged(client, cm):  # pylint: disable=invalid-name
-    """Create a bucket, mark logged, and see that /all/ shows both entries."""
-    client.login(username='test', password='test')
-    buckets = (cm.create_bucket(shortDescription="bucket #1"),
-               cm.create_bucket(shortDescription="bucket #2", bug=cm.create_bug('123')))
-    response = client.get(reverse("crashmanager:allsignatures"))
-    LOG.debug(response)
-    assert response.status_code == requests.codes['ok']
-    siglist = response.context['siglist']
-    assert len(siglist) == 2  # 2 buckets
-    assert set(siglist) == set(buckets)  # same buckets we created
-    assert_contains(response, "bucket #1")
-    assert_contains(response, "bucket #2")
-    assert_contains(response, ALL_SIGS_FMT % 2)
-
-
-def test_all_signatures_view_toolfilter(client, cm):  # pylint: disable=invalid-name
-    """Check that toolfilter is ignored in /all/."""
-    client.login(username='test', password='test')
-    bucket1 = cm.create_bucket(shortDescription="bucket #1")
-    bucket2 = cm.create_bucket(shortDescription="bucket #2")
-    cm.create_crash(shortSignature="crash #1", tool="tool #1", bucket=bucket1)
-    cm.create_crash(shortSignature="crash #2", tool="tool #2", bucket=bucket1)
-    cm.create_crash(shortSignature="crash #3", tool="tool #1", bucket=bucket1)
-    cm.create_crash(shortSignature="crash #4", tool="tool #1", bucket=bucket2)
-    cm.create_toolfilter("tool #1")
-    response = client.get(reverse("crashmanager:allsignatures"))
-    LOG.debug(response)
-    assert response.status_code == requests.codes['ok']
-    siglist = response.context['siglist']
-    LOG.debug(siglist)
-    assert len(siglist) == 2  # 2 buckets
-    assert siglist[0] == bucket2
-    assert siglist[0].size == 1
-    assert siglist[1] == bucket1
-    assert siglist[1].size == 3
-    assert_contains(response, "bucket #1")
-    assert_contains(response, "bucket #2")
-    assert_contains(response, ALL_SIGS_FMT % 2)
+    assert_contains(response, 'signatureslist')
```

Comment:
Please add a unit test for the new serializer, simply calling the API with `?vue=1` should give a different output